src/index.js

```diff
@@ -26,40 +27,119 @@ type Options = {
     onResponseNotFound?: (r: Request) => mixed,
 };
 
+const CONTENT_TYPE_MULTIPART_RE = /^multipart\/(?:form-data|related)(?:;|$)/i;
+const CONTENT_TYPE_FORM_URLENCODED_RE = /^application\/x-www-form-urlencoded(?:;|$)/i;
+
+const isMultipart = (contentType) => CONTENT_TYPE_MULTIPART_RE.exec(contentType);
+const isFormUrlencoded = (contentType) => CONTENT_TYPE_FORM_URLENCODED_RE.exec(contentType);
+
 const createServer = (options?: Options): Server => {
     const {port = 0, ssl, onResponseNotFound} = options || {};
     const mockingServer = createMockingServer({onResponseNotFound});
 
+    const sendResponse = (response, headers, code, content) => {
+        response.statusCode = code;
+        Object.keys(headers).forEach((h) => response.setHeader(h, headers[h]));
+        response.write(content);
+        response.end();
+    };
+
     const handleRequest = (request, response) => {
-        const form = new multiparty.Form();
-        form.parse(request, (err, formFields = {}, files) => {
-            const {method, url, headers} = request;
-            const {pathname, query = {}} = parseUrl(url, true);
-
-            if (pathname === '/__admin__/kill') {
-                response.write('killed');
-                response.end(() => {
-                    process.exit();
-                });
-            }
+        const {method, url, headers} = request;
+        const {pathname, query = {}} = parseUrl(url, true);
+        const contentType = request.headers['content-type'];
+
+        if (pathname === '/__admin__/kill') {
+            response.write('killed');
+            response.end(() => {
+                process.exit();
+            });
+        }
 
+        if (method === 'GET') {
             const res = mockingServer.handle({
                 method,
                 urlPath: pathname || '',
                 urlParams: query,
                 headers,
-                formFields,
+                formFields: {},
             });
 
             if (res) {
-                const {headers: responseHeaders, content, statusCode} = res;
-
-                response.statusCode = statusCode;
-                Object.keys(responseHeaders).forEach((h) => response.setHeader(h, responseHeaders[h]));
-                response.write(content);
-                response.end();
+                return sendResponse(response, res.headers, res.statusCode, res.content);
             }
-        });
+        }
```

Comment:
I miss an else here
```js
} else if (isMultipart(contentType)) {
```

Reply:
yep, I made an early return if the request is a GET, but I could add an else here if you prefer